src/dmd/semantic2.d

```diff
@@ -75,8 +75,10 @@ enum LOG = false;
  */
 extern(C++) void semantic2(Dsymbol dsym, Scope* sc)
 {
+    import dmd.timetrace_sema;
     scope v = new Semantic2Visitor(sc);
-    dsym.accept(v);
+    scope vtimetrace = new SemanticTimeTraceVisitor!Semantic2Visitor(v);
+    dsym.accept(vtimetrace);
```

Comment:
This design was chosen to be able to non-invasively add tracing of the frontend. Now that tracing becomes part of the frontend, a different design choice could be made (e.g. add tracing code directly in SemanticXVisitor), perhaps later.